pkgs/top-level/all-packages.nix

```diff
@@ -29260,7 +29260,7 @@ with pkgs;
   netcoredbg = callPackage ../development/tools/misc/netcoredbg { };
 
   ncdu = callPackage ../tools/misc/ncdu { };
-  ncdu_2 = callPackage ../tools/misc/ncdu_2 { };
```

Comment:
can we keep `ncdu_2` and have `ncdu = ncdu_2`? this would not invalidate the config of people having ncdu_2

Reply:
If, please move it to aliases.

Reply:
Added an alias.